src/garage/tf/samplers/off_policy_vectorized_sampler.py

```diff
@@ -98,7 +99,13 @@ def obtain_samples(self, itr, batch_size):
                 actions, agent_infos = self.algo.policy.get_actions(
                     obs_normalized)
 
-            next_obses, rewards, dones, env_infos = self.vec_env.step(actions)
+            if (isinstance(self.env.action_space, Box)):
```

Comment:
Can we have an option not to scale them?

Reply:
Yes we can. Are there some cases where it's better for the policy to learn the exact unnormalized output values?

Reply:
It is very easy for people to ignore the scaling if we put this in a sampler. Once the policy is trained, if scaling is ignored, the visualization will never be correct and will be confusing. I would suggest either putting this as an option of the policy or just have an environment wrapper to do the job.

Reply:
Ok. Made the scaling an option.